app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.example.readersdk">
```

Comment:
This becomes the `namespace` in build.gradle, right?

Reply:
Yep correct.